build-info-extractor-go/src/main/java/org/jfrog/build/extractor/go/extractor/GoZipBallStreamer.java

```diff
@@ -349,4 +364,29 @@ private String getFirstPathElement(String path) {
         path = StringUtils.removeStart(path, "/");
         return StringUtils.substringBefore(path, "/");
     }
+
+    /**
+     * Sets the Go version based on the content of the go.mod file in the root of the zip file.
+     */
+    private void setGoVersionByModFile(ZipArchiveEntry entry) {
+        // Go version 1.12.0 is the default version where we should start looking for go.mod file.
```

Comment:
It actually go 1.13 where mod files became mandatory
and go 1.11 where mod file became experimental